app/models/manageiq/providers/kubernetes/container_manager/entities_mapping.rb

```
@@ -22,4 +22,12 @@ module ManageIQ::Providers::Kubernetes::ContainerManager::EntitiesMapping
   def miq_entity(entity)
     MAPPING[entity]
   end
+
+  # NOTE: Use of this method may result in unexpected behavior.  If more than
+  # one ManageIQ class maps to an entity, this method will only return the first
+  # instance.  For example, ContainerProject maps to 'namespace' and 'project'.
+  # This method will only return 'namespace'
```

I wasn't thinking of changing the hash, I was thinking on having the reverse behavior return a list - that seems more extensible.

I also expect that we will need to know if resource X is from kubernetes or openshift
(I think currently in some template creation code we try both clients and swallow failures)

I tried to figure out the usage of this by searching in the depending PRs, but could not find any.
Can you provide a link OR a code snippet for your usage of this?
Thanks!